src/components/Box/__tests__/Box.ssr.spec.jsx

```diff
@@ -0,0 +1,16 @@
+/**
+ * @jest-environment node
+ */
+
+import React from 'react'
+import { render } from 'enzyme'
+
+import Box from '../Box'
+
+describe('responsive behaviour', () => {
```

Comment:
I think this describe can be a bit more specific so that it will show up properly in the output. The behaviour being tested is also about SSR I believe, not responsiveness.

Perhaps it should be:

```js
describe('Box server side rendering', ...)
```